src/main/java/de/tum/in/www1/artemis/domain/notification/GroupNotificationFactory.java

```diff
@@ -72,49 +79,67 @@ public static GroupNotification createNotification(Exercise exercise, User autho
             String notificationText) {
         String title;
         String text;
+        boolean textIsPlaceholder;
+        String[] placeholderValues;
         NotificationPriority priority = MEDIUM;
 
         switch (notificationType) {
             case EXERCISE_RELEASED -> {
                 title = EXERCISE_RELEASED_TITLE;
-                text = "A new exercise \"" + exercise.getTitle() + "\" got released.";
+                text = NotificationTitleTypeConstants.EXERCISE_RELEASED_TEXT;
```

Comment:
```suggestion
                text = EXERCISE_RELEASED_TEXT;
```
It should work this way.

Reply:
Unfortunately, it doesn't work like this because there is a constant with the same name in the notificationTargetFactory that also gets imported. So I have to clarify which one should be used.